activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Allow overriding SQLite defaults from `database.yml`
+
+    Any PRAGMA configuration set under the `pragmas` key in the configuration file take precedence over Rails' defaults, and additional PRAGMAs can be set as well.
+
+    ```yaml
+    database: storage/development.sqlite3
+    timeout: 5000
+    pragmas:
+      journal_mode: off
```

Comment:
A nit, but I wouldn't include `journal_mode: off` as an example here.   Someone is bound to copy this into their database config as an example and will inadvertently be left without journaling on their db.  Instead `journal_mode: wal` is probably better as that's most likely the intention of the developer when they want to configure the pragmas.    If you need to push a change to fix the build this might be one to consider.

Reply:
What journal mode do you need for your read-only database? And maybe for the example we should choose a different PRAGMA altogether to demonstrate overriding a default; which one feels best to override in an example in the CHANGELOG to your mind?